primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -2719,8 +2743,11 @@
             }
 
             var $this = this;
-            $(window).on('resize.' + this.options.id, function() {
-                $this.hideOverlay();
+            $(window).on('resize.' + $this.options.id, function() {
+                // TODO: lazy model for some reason triggers a windows resize
```

Comment:
@jepsar @christophs78 something to look at in the future but DatePicker012 test was failing and i debugged and dumped JS stacks to figure out for some reason window resize is called but only in that specific 012 Lazy scenario. Its like the redrawing of the lazy panel somehow triggers a window resize i could not figure out but didn't want to hold up this PR as it fixes a bunch of accessibility issues.